server/enums/workInterestTypeValue.ts

```diff
@@ -16,4 +16,4 @@ enum WorkInterestsValue {
   OTHER = 'OTHER',
 }
 
-export default WorkInterestsValue
+export default WorkInterestTypeValue
```

Comment:
Changing the name of this enum has made the diff quite large.

Reply:
Good idea for changing it though - makes it consistent with the other "type" enums 👍